WooCommerce/WooCommerceTests/Authentication/Jetpack Setup/LoginJetpackSetupViewModelTests.swift

```diff
@@ -176,7 +176,7 @@ final class LoginJetpackSetupViewModelTests: XCTestCase {
 
         // Then
         XCTAssertTrue(viewModel.shouldShowInitialLoadingIndicator)
-        waitUntil(timeout: 1) {
+        waitUntil {
```

Comment:
Thanks for working on this, @iamgabrielma ❤️ 

To fix the flakiness, I think we should remove the `0.3` delay. 🤔 

Maybe we can break the test into two.
1. test_shouldShowInitialLoadingIndicator_turns_on_correctly
In this we can avoid calling the success completion block and test  that`viewModel.shouldShowInitialLoadingIndicator` is true
2. test_shouldShowInitialLoadingIndicator_turns_off_correctly
Call the success completion block and test that `viewModel.shouldShowInitialLoadingIndicator` is false

@itsmeichigo What do you think?

Reply:
@selanthiraiyan's suggestion sounds good to me 👍

Reply:
Sounds good! I split the test as well as `test_shouldShowSetupSteps_is_correct` (which also fails sometimes due to the same reason) into two on https://github.com/woocommerce/woocommerce-ios/pull/8452/commits/1d95d03d263ef546b9c804d809abb901320edc07 . Ready for re-review @selanthiraiyan 🙇